app/assets/javascripts/pulitzer.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function(){
```

Comment:
@gomayonqui I believe this whole thing can be replaced with a Hooch Revealer. I haven't added Revealers to the Hooch wiki yet, I'll work on that today.